views/hr.handlebars

```diff
@@ -69,14 +52,25 @@
     </div>
 </div>
 
-<div id="modal8" class="modal" style="background-color: transparent;">
+<div id="modal8" class="modal indigo lighten-3 bottom-sheet">
```

Comment:
If you don't want it to be a bottom-sheet style modal, just remove the `bottom-sheet` class from the above.